locale/locales/en.json

```
@@ -31,6 +31,34 @@
     "value": "Last page",
     "comment": "tooltip and aria-label for the last page button. (tew 210929)"
   },
+  "SNTable.Pagination.MovedToFirstPage": {
+    "value": "You are in the first page.",
```

```suggestion
    "value": "You are on the first page.",
```

---

and same for the rest

---

Actually these should be the same as `MovedToNextPage`
```suggestion
    "value": "Moved to the first page.",
```

---

✅ fixed